ci/terraform/account-management/remove-account.tf

```diff
@@ -23,7 +23,7 @@ module "delete_account" {
   security_group_id                      = aws_vpc.account_management_vpc.default_security_group_id
   subnet_id                              = aws_subnet.account_management_subnets.*.id
   environment                            = var.environment
-  lambda_role_arn                        = aws_iam_role.dynamo_sqs_lambda_iam_role.arn
+  lambda_role_arn                        = module.account_notification_dynamo_sqs_role.arn
```

Comment:
Have you tried this in the sandpit ... I've seen issues historically with inputs of one module being reliant on outputs from another (particuarly when bootstrapping)!?

Reply:
This applies cleanly in sandpit